src/frontend-samples/fire-decoration-sample/README.md

```diff
@@ -0,0 +1,30 @@
+# Particle Effect - Fire
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample shows how to create a particle effect using fire as an example.
+
+## Purpose
+
+The purpose of this sample is to demonstrate the following:
+
+* Creating a decorator and rendering it in the active view.
+* Working with particle effects using the [ParticleCollectionBuilder](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/particlecollectionbuilder) API.
+* The visualizing (NOT simulating) fire in an iModel.
+
+## Description
+
+This sample provides an example of a particle effect in world space (as opposed to view space).  The sample provides features to place and configure fire particle emitters.  Each emitter is initially configured based on preset parameters defined in [FireDecorationApp.ts](./FireDecorationApp.tsx).  Select which preset is used with the drop down menu.  This sample does not demonstrate a way to configure an emitter after it has been deselected.
+
+For a basic decorator, the [Decorator](https://www.itwinjs.org/reference/imodeljs-frontend/views/decorator/) interface is implemented. The decorate method adds the [RenderGraphics](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/rendergraphic/) to the given [DecorateContext](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/decoratecontext). Once the decorator is created, pass it in the [ViewManager.addDecorator](https://www.itwinjs.org/reference/imodeljs-frontend/views/viewmanager/adddecorator/) method to have it rendered in all active views. This sample is specifically for working with particle effects. For examples of other decorators, see [Geometry Samples](https://www.itwinjs.org/sample-showcase/?group=Geometry+Samples&sample=simple-line-sample) and this sample's [HighlightEmitter](./FireDecorator.ts).
```

Comment:
Refer to the Heatmap decorator sample rather than the geometry samples.  The geometry samples are to demonstrate geometry concepts and only use a decorator where heatmap is all about making a custom decorator.